### PR TITLE
Remove option2 from documentation page

### PR DESCRIPTION
**Pull request by JavierJia**

### CODE REVIEW

File: docs/documentation.md [hunk 1]
```diff
@@ -309,3 +307,10 @@ Sometimes the front-end wants to slice multiple queries simultaneously so that i
   }
 }
 ```
+
+### Advanced users
+
+Real-world workloads may require a multi-nodes AsterixDB cluster.
```
Comment (chenlica) [inline docs/documentation.md]: "Real-world workloads" -> "Some applications".
"multi-nodes" -> "multi-node".

File: docs/documentation.md [hunk 2]
```diff
@@ -309,3 +307,10 @@ Sometimes the front-end wants to slice multiple queries simultaneously so that i
   }
 }
 ```
+
+### Advanced users
+
+Real-world workloads may require a multi-nodes AsterixDB cluster. 
+You can follow the official [documentation](https://ci.apache.org/projects/asterixdb/install.html) to setup up it.
```
Comment (chenlica) [inline docs/documentation.md]: "setup up it" -> "set it up"

File: docs/documentation.md (hunk 1 above)
Comment (JavierJia) [inline docs/documentation.md]: done

File: docs/documentation.md (hunk 2 above)
Comment (JavierJia) [inline docs/documentation.md]: done

**Pull request merged**